Extensions/PlatformBehavior/tests/platformerobjectruntimebehavior.spec.js

```diff
@@ -1689,7 +1700,9 @@ describe('gdjs.PlatformerObjectRuntimeBehavior', function () {
       object.getBehavior('auto1').simulateLadderKey();
       climbLadder(10);
       stayOnLadder(10);
-      climbLadder(14);
+      releaseLadder(10);
```

Comment:
Nitpicking: could be interesting to add the `expect(object.getY()).to.be.within(` before the releaseLadder and after the 10 frames of releaseLadder.

(You can take the numbers given by the test itself, even if in theory we can compute them, the final position after the 10 frames being the initial Y position after `stayOnLadder(10)`, and 10 frames of descent that we could calculate using the given gravity).

This would robustify a bit more the test (we can imagine that in the future, we would have a bug where if you keep pressing the release ladder button, the object is stuck in the air or on the contrary get the gravity applied multiple time, etc...)